config/i18n-tasks.yml

```diff
@@ -1,7 +1,7 @@
 # i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
 
 # The "main" locale.
-base_locale: en
+base_locale: es
```

Comment:
this change is for development only right? we probably want to default to `en` still?